pyke/lightcurve.py

```diff
@@ -93,12 +93,37 @@ def flatten(self, window_length=101, polyorder=3, **kwargs):
     def fold(self, phase, period):
         return LightCurve(((self.time - phase + 0.5 * period) / period) % 1 - 0.5, self.flux)
 
-    def draw(self):
-        raise NotImplementedError("Should we implement a LightCurveDrawer class?")
-
     def to_csv(self):
         raise NotImplementedError()
 
+    def plot(self, ax=None, norm=True, xlabel = 'Time - 2454833 (days)',
+            ylabel = 'Normalised Flux', title=None, color='#363636', fill=False, grid=True, legend=True, **kwargs):
+        """
+        Plot a Light Curve.
```

Comment:
We should explain the attributes and return value here.

Reply:
Added a docstring